### PR TITLE
Upgrade sdk & extension api

### PR DESCRIPTION
**Pull request by AminTi**

### CODE REVIEW

File: src/confighub/components/Header.tsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ const Header = ({ title, status, onSaveClick }: IHeaderProps) => {
       </HeaderTitleArea>
       <HeaderSideContainer>
         <Button
-          text='Save configuration'
+          text='Save configuration Test'
```
Comment (madkoo) [inline src/confighub/components/Header.tsx:41]: Probably there shouldnt be "test" here ?

**Review by madkoo — COMMENTED** [review]
Check my comments

File: src/confighub/components/Header.tsx (hunk 1 above)
Comment (AminTi) [reply src/confighub/components/Header.tsx:41]: done

**Review by madkoo — APPROVED** [review]
Looks fine to me

**Pull request merged**